defs.bzl

```
@@ -46,7 +46,13 @@ def _shader_binary(ctx):
         arguments = glslcArgs,
     )
 
-    return DefaultInfo(files = depset([spvOut]))
+    runfiles = ctx.runfiles(collect_data = True)
+    runfiles = runfiles.merge(ctx.runfiles(files = [spvOut]))
+
+    return [DefaultInfo(
+        files = depset([spvOut]),
+        runfiles = runfiles,
+    )]
```

Is the `collect_data` necessary? Documentation says it's not recommended and all that is needed is the spv file in the runfiles, correct?

```suggestion
    return DefaultInfo(
        files = depset([spvOut]),
        runfiles = ctx.runfiles(files = [spvOut]),
    )
```

---

I think this will not include the runfiles from any dependencies of the `shader_binary` rule, but I doubt that matters.